vslib/src/SwitchStateBase.cpp

```diff
@@ -1783,6 +1783,10 @@ sai_status_t SwitchStateBase::refresh_read_only(
             case SAI_SWITCH_ATTR_AVAILABLE_DNAT_ENTRY:
             case SAI_SWITCH_ATTR_AVAILABLE_DOUBLE_NAT_ENTRY:
                 return SAI_STATUS_SUCCESS;
+
+            case SAI_SWITCH_ATTR_NUMBER_OF_SYSTEM_PORTS:
+            case SAI_SWITCH_ATTR_SYSTEM_PORT_LIST:
+                return SAI_STATUS_SUCCESS;
```

Comment:
this is wrong, i dont see actual list to br refreshed here or ports

Reply:
This is needed. PortsOrch reads this read-only attributes (SWITCH object). These are all fixed one time static configuration. They are not changed by SAI. No refreshing is needed. But returning failure will return failure for GET. So we always return SUCCESS.